ci/run.sh

```diff
@@ -8,8 +8,15 @@ if [ "$1" = "aarch64-apple-darwin" ] ; then
   export CARGO_TARGET_AARCH64_APPLE_DARWIN_RUNNER=echo
 fi
 
-cargo test --manifest-path "$testcrate_dir/Cargo.toml" --target $1 -vv
-cargo test --manifest-path "$testcrate_dir/Cargo.toml" --target $1 -vv --release
+if [ "$1" = "wasm32-wasi" ] ; then
```

Comment:
Could this move from the `run.sh` script to a dockerfile where the platform-specific runners and such live?

Reply:
Yep! Seeing the WASI CI test fail, I started doing exactly that about the *test dependencies*. 
Adding to that, should I just pull out the `wasm32-wasi` target from the run.sh matrix instead and as you say, make it fully self-contained in a `Dockerfile`?